### PR TITLE
Fix cypress dropdown search & Refactor cypress structure

### LINKED ISSUE #124: Fix Cypress tests to be able to properly find elements
There are problems with:

- Radio buttons
- Dropdowns

### PR DESCRIPTION
**Pull request by GodVenn**
Closes #124

### CODE REVIEW

File: frontend/src/Order.tsx [hunk 1]
```diff
@@ -178,30 +190,26 @@ const Order = () => {
...
+                    <Grid item container xs={12} spacing={3} alignItems="center">
+                        <Grid item xs={10} sm={5} data-testid={'user_type_dropdown'}>
+                            <FieldHeader headerText={'User type'} />
+                            <Select options={userTypeOptions} onSelect={item => setSingleField('userType', item.title)} />
+                        </Grid>
+                        <HelpIcon helpText={'info text'} />
                     </Grid>
                     {userType === 'Equinor personnel' ? (
                         //Equinor employee device
                         <>
```
Comment (VegardOztan) [inline frontend/src/Order.tsx:212]: We don't need the parent `<></>` anymore as the outer `Grid `contains all element

File: frontend/src/Order.tsx [hunk 2]
```diff
@@ -224,27 +232,20 @@ const Order = () => {
                     ) : (
                         //External employee device
                         <>
```
Comment (VegardOztan) [inline frontend/src/Order.tsx:234]: Same as prev. comment

**Review by VegardOztan — CHANGES_REQUESTED** [review]

File: frontend/cypress/page_objects/order_page.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        return cy.get('[data-testid=ok_button]')
+    }
+
+    getConfirmButton = () => {
+        return cy.get('[data-testid=confirm_button]')
+    }
+
+    getCancelButton = () => {
+        return cy.get('[data-testid=cancel_button]')
+    }
```
Comment (maiaht) [inline frontend/cypress/page_objects/order_page.ts:50]: Could these buttons be called something more specific?

File: frontend/cypress/page_objects/order_page.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+    getShortnamesInputField = () => {
+        return cy.get('[data-testid=shortname_input]')
+    }
+
+    getiPadAmountInputField = () => {
+        return cy.get('[data-testid=ipad_amount_input]')
+    }
+
```
Comment (maiaht) [inline frontend/cypress/page_objects/order_page.ts:39]: Missing function to get SIM radio buttons

File: frontend/cypress/page_objects/order_page.ts [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    getSubmitButton = () => {
+        return cy.get('[data-testid=submit_button]')
+    }
+
+    getSubmitDialog = () => {
+        return cy.get('[data-testid=submit_dialog]')
+    }
+
+    getCountWarningDialog = () => {
```
Comment (maiaht) [inline frontend/cypress/page_objects/order_page.ts:60]: Rename to `getAmountWarningDialog`?

**Review by maiaht — CHANGES_REQUESTED** [review]

File: frontend/cypress/page_objects/order_page.ts (hunk 3 above)
Comment (GodVenn) [reply frontend/cypress/page_objects/order_page.ts:50]: Not sure, does the testid have to be unique? 
In BMT this is how they are named as well, despite their generic names

File: frontend/cypress/page_objects/order_page.ts (hunk 5 above)
Comment (GodVenn) [reply frontend/cypress/page_objects/order_page.ts:60]: Aw SHUCKS, missed one.

File: frontend/cypress/page_objects/order_page.ts (hunk 3 above)
Comment (maiaht) [reply frontend/cypress/page_objects/order_page.ts:50]: I was thinking about the function names. For instance, using `orderPage.getOkButton()` does not indicate what is OK if you press this button. So maybe change the function names to more descriptive names?

Comment (GodVenn) [reply frontend/cypress/page_objects/order_page.ts:50]: But what if there is a possibility for more than one ok button, if we have more popups further down the line?

**Review by maiaht — APPROVED** [review]

**Review by VegardOztan — APPROVED** [review]

**Pull request merged**